ts/sdk/package.json

```diff
@@ -11,7 +11,7 @@
   },
   "dependencies": {
     "@coral-xyz/anchor": "^0.26.0",
-    "@drift-labs/sdk": "2.42.0-beta.17",
+    "@drift-labs/sdk": "file:../../../drift-common/protocol/sdk",
```

Comment:
is this a temp fix? asking because technically this repo is open sourced, so not relevant to everyone else

Reply:
agree this should be removed

Reply:
oh I didn't know that .. sry .. will take it back